### DISCUSSION TITLE
Anyone interested in joining a discord chat with a group of people that actively work in the medical imaging field?

### DISCUSSION THREAD

**Discussion by dnoodles** [fo-dicom/fo-dicom#1714] [General]
Hey everyone, I am the owner of a consulting company specializing in medical imaging IT.  I do alot of development using fo-dicom to create custom migration scripts.  I would live to grow my community of fo-dicom users as well as anyone that uses open source tools such as pydicom, orthanc and mercur DICOM orchestrator.  Feel free to join my chat below:

https://discord.gg/EzkmkSPt

**Comment by sourouronally** [comment]
Hi , this discord link is unavailale , could you please verify and share with us the correct link , thank you !

**Reply by dnoodles** [reply]
https://discord.gg/nxyHsaR3 

good for 7 days.
